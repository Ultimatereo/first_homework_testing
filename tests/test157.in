4
1.58e-05 3.36e-05 2.69e-05 3.79e-05 2.04e-05
4.07e-05 3.37e-05 7e-06 4.3e-06 1.85e-05
1.16e-05 9.2e-06 1.57e-05 9.9e-06 1e-07
4.23e-05 1.27e-05 2.99e-05 4.14e-05 1.64e-05
