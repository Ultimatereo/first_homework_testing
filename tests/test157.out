0.177202
0.413328
-0.775696
0.648512
